### PR TITLE
Identity 788 - Informix error while adding a tenant

### PR DESCRIPTION
**Pull request by cdwijayarathna**

### CODE REVIEW

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java [hunk 1]
```diff
@@ -718,10 +718,10 @@ private void setBlobValue(String value, PreparedStatement prepStmt, int index) t
             if (inputStream != null) {
                 prepStmt.setBinaryStream(index, inputStream, inputStream.available());
             } else {
-                prepStmt.setBinaryStream(index, inputStream, 0);
+                prepStmt.setBinaryStream(index, new ByteArrayInputStream(CharacterEncoder.getSafeText("").getBytes()), 0);
```
Comment (Pushpalanka) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Adding the charset name when getting bytes would have been better to make this platfrom independent. Also do we really need to getSafeText on empty String?

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java [hunk 2]
```diff
@@ -718,10 +718,10 @@ private void setBlobValue(String value, PreparedStatement prepStmt, int index) t
...
             }
         } else {
-            prepStmt.setBinaryStream(index, null, 0);
+            prepStmt.setBinaryStream(index, new ByteArrayInputStream(CharacterEncoder.getSafeText("").getBytes()), 0);
```
Comment (Pushpalanka) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Adding the charset name when getting bytes would have been better to make this platfrom independent. Also do we really need to getSafeText on empty String?

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java (hunk 1 above)
Comment (cdwijayarathna) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Thanks for pointing, Fixed with https://github.com/wso2/carbon-identity/pull/1675

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java (hunk 2 above)
Comment (cdwijayarathna) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Thanks for pointing, Fixed with https://github.com/wso2/carbon-identity/pull/1675

**Pull request merged**